application-forum-ui/src/main/resources/ForumCode/ForumSheet.xml

```diff
@@ -55,14 +55,15 @@
   #elseif($xcontext.action == 'view')
     #initForum($spaceName)
     ## Depending on XE version, we consider parent-child relationship for Pre NS or space-subspace for NS.
-    #set($hierarchyCondition = "doc.parent=:forumDoc or doc.space=:forumSpace")
-    #set($topics = $services.query.xwql("from doc.object(ForumCode.TopicClass) as topic where $hierarchyCondition").bindValue('forumDoc', $doc.fullName).bindValue('forumSpace', $doc.space).execute())
+    #set($hierarchyCondition = 'doc.parent=:parent or doc.space=:space')
+    #set($topics = $services.query.xwql("from doc.object(ForumCode.TopicClass) as topic where $hierarchyCondition").bindValue('parent', $doc.fullName).bindValue('space', $doc.space).execute())
   ## Count comments
```

Comment:
Yep, on a quick look it seems like indeed, the `$comments` variable is not used anywhere and the entire `## Count comments` section could be removed.
Seems to be a leftover coming from the initial commit: https://github.com/acotiuga/application-forum/commit/75d4cb8e61896bef7c671b4bd07cf454240f73dd.